src/main/java/com/moorhuhnservice/moorhuhnservice/controller/ConfigController.java

```diff
@@ -0,0 +1,94 @@
+package com.moorhuhnservice.moorhuhnservice.controller;
+
+import com.moorhuhnservice.moorhuhnservice.data.*;
+import com.moorhuhnservice.moorhuhnservice.data.mapper.ConfigurationMapper;
+import com.moorhuhnservice.moorhuhnservice.data.mapper.QuestionMapper;
+import com.moorhuhnservice.moorhuhnservice.repositories.ConfigurationRepository;
+import com.moorhuhnservice.moorhuhnservice.service.ConfigService;
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v1/minigames/moorhuhn/configurations")
+@Slf4j
+public class ConfigController {
+
+  @Autowired
+  ConfigurationRepository configurationRepository;
+
+  @Autowired
+  ConfigService configService;
+
+  @Autowired
+  QuestionMapper questionMapper;
+
+  @Autowired
+  ConfigurationMapper configurationMapper;
+
+  @GetMapping("")
+  public List<ConfigurationDTO> getConfigurations() {
+    log.debug("get all configurations");
+    List<Configuration> configurations = configurationRepository.findAll();
+    List<ConfigurationDTO> configurationDTOs = configurationMapper.configurationsToConfigurationDTOs(configurations);
+    return configurationDTOs;
```

Comment:
```suggestion
    return configurationMapper.configurationsToConfigurationDTOs(configurationRepository.findAll());
```

Reply:
I am responding here for all things you marked like this.
I have consistently remap into a variable before I returned it. When I do it like you suggest it (remap it on return) what difference does it make?

Reply:
That we don't need "unused" variables.
I count a variable as unused when it is only used once.
For me, that improves the readability of the code flow drastically.

Reply:
> That we don't need "unused" variables. I count a variable as unused when it is only used once. For me, that improves the readability of the code flow drastically.

I can not agree with this. I can understand that the mapping can be returned and for this is now onw declaration needed, but for the `configurationRepository.findAll()` must be an own declaration, this improves readability a lot in my eyes.

Reply:
Because most team members voted for this solution in internal chat we apply these changes